beta/next.config.js

```diff
@@ -17,14 +17,16 @@ module.exports = {
   async redirects() {
     return redirects.redirects;
   },
-  rewrites() {
-    return [
-      {
-        source: '/feed.xml',
-        destination: '/_next/static/feed.xml',
-      },
-    ];
-  },
+  // TODO: this causes extra router.replace() on every page.
```

Comment:
See here https://github.com/vercel/next.js/pull/33861#issuecomment-1135071203.

Originally I thought the `startTransition` call worked around it, but I was still seeing the issue. I ended up just ripping out this feature for now to avoid the false positive router navigations.